starlette_web/contrib/auth/views.py

```diff
@@ -204,6 +206,9 @@ async def _validate(self, request, *_) -> dict:
         if email != user_invite.email:
             raise InvalidParameterError(message="Email does not match with your invitation.")
 
+        _user = User(email=cleaned_data["email"])
+        validate_password(cleaned_data["password_1"], _user)
```

Comment:
тут юзер используется только для валидации. можем сразу передать инстанс?
```suggestion
        validate_password(cleaned_data["password_1"], User(email=email))
```

Reply:
готово